scripts/generate_gff3_db.py

```diff
@@ -0,0 +1,127 @@
+"""
```

Comment:
Note that the way I wrote it, the GFF3 to DB converter can be used either as a standalone script _or_ as a library within another script. I'm using it in the latter mode in the `prepare_compliance_data.py` script, on line 168.